docs/VnVPlan/VnVPlan.tex

```diff
@@ -310,6 +310,83 @@ \subsubsection{Area of Testing2}
+
+Initial State: User is not logged in or terms are not accepted.
+                
+Input: User tries to access the application's database.
+                
+Output: Access will be denied until the user accepts terms and services.
+                
+How test will be performed: Attempts will take place to access database without accepting terms. The team will confirm denial of access. Then, attempt access after accepting terms and services. The team will confirm successful database access.
+
+\item{NFRT-SR2\\}
+
+Type: Functional, Dynamic
+
+Initial State: Data is stored in the application database.
+                
+Input: User will attempt to modify the data directly from the application database.
+                
+Output: Stored data are encrypted. Therefore, modifications to the data are denied.
+                
+How test will be performed: The team will attempt to use database tools to try to modify data within the application database. Then, the team will confirm that the application backend database is secure and has not been modified.
+
+\item{NFRT-SR3\\}
+
+Type: Dynamic, Penetration
+
+Initial State: Application is in operation.
+                
+Input: User will attempt simulated attacks like SQL injection, cross-site scripting, etc.
+                
+Output: Application will remain secure without any data breaches.
+                
+How test will be performed: The team will conduct a series of different types of simulated cyber attacks. The team will then observe application response and monitor application database.
+
+\item{NFRT-SR4\\}
+
+Type: Functional, Dynamic, Manual
+
+Initial State: Raw user data is stored in the application database.
+                
+Input: The raw user data is processed and then stored in the application database.
+                
+Output: The data is anonymized by data perturbation (inducing small amount of noise) and encryption. The data is also sanitized by data deletion or masking.
+                
+How test will be performed: The team will inspect the stored user data for any personal identifiers. The team will search for any presence of accessible sensitive user data.
+
+\item{NFRT-SR5\\}
+
+Type: Dynamic, Manual
+
+Initial State: Application has yet to collect user data.
+                
+Input: User will start using the application.
+                
+Output: A consent form will pop up before any user data is collected. No user data will be collected without explicit consent given by user.
+                
+How test will be performed: The team will check for prompts that ask users to consent to data collection. The team will then test the application with and without user consent to ensure compliance. The team will confirm no user data was collected and stored in the database without user consent.
+
+\item{NFRT-SR6\\}
+
+Type: 
+
+Initial State: Application is in operation.
+                
+Input: User will attempt various actions to induce system events.
```

Comment:
This seems pretty vague/ambiguous. Can you make this more targeted?

Reply:
resolved